src/util/setGenotype.js

```diff
@@ -95,5 +100,5 @@ export const orderOfGenotypeAppearance = (nodes, aaGenotype) => {
   if (ordered.indexOf("X") !== -1) {
     orderedBases.push("X");
   }
-  return orderedBases;
-};
+  return orderedBases
+}
```

Comment:
probably want to keep the `;` on the `return` expression?

Reply:
Thanks - fixed up